rpmlint/checks/BinariesCheck.py

```diff
@@ -137,9 +135,9 @@ def _check_shared_library(self, pkg, pkgfile_path, path):
             # libs only, but that has potential to generate lots of
             # false positives and noise.
             for symbol in self.ldd_parser.undefined_symbols:
-                self.output.add_info('W', pkg, 'undefined-non-weak-symbol', path, symbol)
+                self.output.add_info('E', pkg, 'undefined-non-weak-symbol', path, symbol)
```

Comment:
why this change ?

Reply:
Should've been error previously missed it in previous commit. Could've added it as separate commit.